examples/SimpleCalc.py

```diff
@@ -32,7 +32,7 @@
 
 variables = {}
 
-from fourFn import BNF, exprStack, evaluate_stack
+from examples.fourFn import BNF, exprStack, evaluate_stack
```

Comment:
All examples must be run from the main pyparsing directory, not from the `examples` directory